<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>333</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Распознавание ГРЗ ТС КНР</string>
  </property>
  <property name="locale">
   <locale language="Russian" country="Russia"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="gvImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>391</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoadImage">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать...</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tbImagePath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblImagePath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Путь к файлу изображения:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>251</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Загруженное изображение:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRecognitionResult">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Распознанный иероглифический символ:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tbRecognitionResult">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRecognize">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Распознать</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="mainMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>410</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="miFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="miOpen"/>
    <addaction name="separator"/>
    <addaction name="miExit"/>
   </widget>
   <widget class="QMenu" name="miUpdate">
    <property name="title">
     <string>Обновление</string>
    </property>
    <addaction name="miCheckUpdate"/>
   </widget>
   <addaction name="miFile"/>
   <addaction name="miUpdate"/>
  </widget>
  <action name="miOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="miExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="miCheckUpdate">
   <property name="text">
    <string>Обновить</string>
   </property>
  </action>
  <action name="miAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="miHowTo">
   <property name="text">
    <string>Инструкция</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
